docs/zookeeperAdmin.html

```diff
@@ -2213,6 +2213,7 @@ <h4>The Four Letter Words</h4>
               zk_min_latency  0
               zk_packets_received 70
               zk_packets_sent 69
+              zk_num_alive_connections	1
```

Comment:
Please remove doc/{html,pdf} from PRs - these files are regenerated during commit time by the committer. (and are a pain to manage wrt the PR merge tool)